### PR TITLE
Update issue templates

### PR DESCRIPTION
**Pull request by awiawi**

### CODE REVIEW

**Comment by awiawi** [comment]
@raphajaner  What do you think about these issue templates for this repository?

File: .github/ISSUE_TEMPLATE/bug_report.md [hunk 1]
```diff
@@ -0,0 +1,32 @@
...
+**To Reproduce**
+Steps to reproduce the behavior:
+1. Go to '...'
+2. Click on '....'
+3. Scroll down to '....'
+4. See error
+
+**Expected behavior**
+A clear and concise description of what you expected to happen.
+
```
Comment (raphajaner) [inline .github/ISSUE_TEMPLATE/bug_report.md:22]:
```suggestion
## Reason and Possible fixes
If you know or suspect the reason for this bug, paste the code lines and suggest modifications.

```

**Review by raphajaner — COMMENTED** [review]
LGTM! What do you think of adding a field so that people can add/suggest a possible fix?

**Review by raphajaner — APPROVED** [review]

**Comment by awiawi** [comment]
I already applied your suggestion. I like it very much. ;-)

**Pull request merged**